### PR TITLE
Added Oliseh as delegate

### PR DESCRIPTION
**Pull request by Olisehgenesis**
# Please Read and check all that apply 

## general 

- [x] no lint or test errors

## If adding delegate

In order to prevent impersonation attacks we ask that the entity to be added as a delegate make a public post either with a link to this pull request or including the same address referenced in the pull request. This post should be on a platform you have already established reputation on. So if you are active on Celo Forum post there or if you have an active Twitter account post there. 

- LINK TO PROOF 

Checklist of common mistakes

- [x] the image is located in the public/logos directory
- [x] the image filename has zero spaces in it
- [x] the image path in the json exactly matches the actual file name

### CODE REVIEW

**Comment by Olisehgenesis** [comment]
https://x.com/illmindofbanana/status/1832433978596630616?s=46
I am really excited about this

**Review by Olisehgenesis — COMMENTED** [review]
All is well 👍

File: src/config/delegates.json [hunk 1]
```diff
@@ -222,4 +233,5 @@
     "interests": ["Staking", "Security", "Passive Income"],
     "description": "Staking is a process of delegating your cryptocurrency to us. Our valdiators are responsible for validating transactions on the blockchain. In exchange for the participation in the network, you will receive a portion of the transaction fees and block rewards that are generated on the network. This means that at the end of each validating cycle (or epoch), your stake increases, generating you passive income."
   }
+
```
Comment (aaronmgdr) [inline src/config/delegates.json:236]:
```suggestion
```

File: src/config/delegates.json [hunk 2]
```diff
@@ -10,6 +10,17 @@
...
+  },
+   "0xa88df1011E9aF811e7dc67396133f69a3CAFf072": {
+    "name": "Oliseh Genesis",
+    "address": "0xa88df1011E9aF811e7dc67396133f69a3CAFf072",
+    "logoUri": "/logos/delegatees/oliseh.jpg",
+    "date": "2024-05-09",
+    "links": {
+      "website": ""
+    },
+    "interests": ["Discovery", "Engaged", "Calm" , "Love"],
```
Comment (aaronmgdr) [inline src/config/delegates.json:22]:
```suggestion
    "interests": ["Discovery", "Engaged", "Calm", "Love"],
```

File: src/config/delegates.json [hunk 3]
```diff
@@ -10,6 +10,17 @@
     },
     "interests": ["Proof-of-Stake", "High Availability", "Infrastructure", "Web3 Investment"],
     "description": "Stake Capital is a trusted and experienced blockchain investment firm, actively contributing to the blockchain ecosystem since 2016. With expertise in venture capital, incubation, acceleration, and validation, we ensure robust participation in governance through our mainnet validation for top projects and our involvement in testnets for emerging infrastructures. Our commitment to excellence and transparency makes us a reliable partner to delegate your voting power, helping to secure and innovate within the Celo network."
+  },
+   "0xa88df1011E9aF811e7dc67396133f69a3CAFf072": {
```
Comment (aaronmgdr) [inline src/config/delegates.json:14]:
```suggestion
  "0xa88df1011E9aF811e7dc67396133f69a3CAFf072": {
```

**Review by aaronmgdr — APPROVED** [review]

**Pull request merged**